vlib/v/help/build/build-c.txt

```diff
@@ -361,3 +361,11 @@ see also `v help build`.
           https://learn.microsoft.com/en-us/cpp/build/reference/fp-specify-floating-point-behavior?view=msvc-170&redirectedfrom=MSDN
           https://clang.llvm.org/docs/UsersManual.html#cmdoption-ffast-math
           https://gcc.gnu.org/onlinedocs/gcc/Optimize-Options.html#index-ffast-math
+
+   -div-by-zero-is-zero
+      Avoids division by zero errors, by treating `x / 0` as being equal to `0`.
+      It also treats `x % 0` as being equal to `x`, because `x%y`==`x-y*(x/y)`,
+      which in turn eliminates another source of division by zero errors.
+      It is mainly useful when prototyping games, and other experimental code with
+      lots of arithmetic expressions, where you do not want to check for the 0 divisor
```

Comment:
`zero` word sounds better than `0`

```suggestion
      lots of arithmetic expressions, where you do not want to check for the zero divisor
```

Reply:
I do not want to trigger all CI jobs just for it.

It can be changed later in another PR (with less checks, since it will affect a single .txt file, unlike here, where there are .v files in the compiler changed too).